src/ontology/edits/oeo-physical.omn

```diff
@@ -6138,20 +6176,27 @@ Class: OEO_00030002
 
 In real world, fossils are from biogenic origin some thousands or millions of years ago. However, this is irrelevant in the energy modelling domain.",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/108
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130
+
+Update definition:
+https://github.com/OpenEnergyPlatform/ontology/issues/872
+https://github.com/OpenEnergyPlatform/ontology/pull/1048",
         rdfs:label "fossil"
     
     SubClassOf: 
         OEO_00030003
     
     DisjointWith: 
-        OEO_00030004
+        
```

Comment:
OEO_00030001

Reply:
It is there (line 6193), but the term tracker item comes first.

Reply:
Ok, looks weird, and I thought it was missing. But if it's there and working like this, ignore my suggestions.